src/shared/directives/substitutor/label-substitutor.directive.ts

```diff
@@ -39,7 +39,18 @@ export class LabelSubstitutorDirective implements OnInit, OnDestroy {
 
   private getReadOnlyAndFormFields() {
     let formFields = this.getFormFieldsValuesIncludingDisabled();
-    return this.fieldsUtils.mergeLabelCaseFieldsAndFormFields(this.eventFields, formFields);
+    let uniqueEventFields = this.removeDuplicates(this.eventFields);
```

Comment:
Please raise a tech debt ticket in JIRA to understand and fix this duplication.

Reply:
Done